Assets/MixedRealityToolkit/Utilities/Editor/ConfigurationCheckers.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
Does the XRSDKConfigurationChecker still need to be moved into the Core? It'd be great if we can keep it in the XRSDK provider folder.

Reply:
it can be moved back. the original thought was it would be needed here eventually 😄